### PR TITLE
add alibaba cloud style on homepage

### PR DESCRIPTION
**Pull request by HYmian**
Fixes # .

Change summary:
- add alibaba cloud style on homepage

Remaining issues / concerns:

### CODE REVIEW

File: src/frontend/handlers.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ var (
 	plat platformDetails
 )
 
-var validEnvs = []string{"local", "gcp", "azure", "aws", "onprem"}
+var validEnvs = []string{"local", "gcp", "azure", "aws", "onprem", "alibaba"}
```
Comment (askmeegs) [inline src/frontend/handlers.go:51]: Could you also please add alibaba as an option here? Thanks! 
https://github.com/GoogleCloudPlatform/microservices-demo/blob/master/kubernetes-manifests/frontend.yaml#L69

**Comment by askmeegs** [comment]
Working on a minikube cluster with `env: alibaba` in frontend.yaml 
Screen Shot 2021-06-20 at 12 54 47 PM

File: src/frontend/handlers.go (hunk 1 above)
Comment (HYmian) [reply src/frontend/handlers.go:51]: done

**Review by askmeegs — APPROVED** [review]

**Comment by HYmian** [comment]
@googlebot I fixed it.

**Comment by HYmian** [comment]
@askmeegs There was an 'Smoke Test' error before, how to fix it？

**Comment by askmeegs** [comment]
> @askmeegs There was an 'Smoke Test' error before, how to fix it？

Hello, could you pull the latest master commit and re-push to your branch? We may have fixed the smoke test issue in the last commit.

**Pull request closed without merging**